Mathlib/CategoryTheory/EssentialImage.lean

```diff
@@ -167,6 +167,11 @@ instance Full.toEssImage (F : C ⥤ D) [Full F] : Full F.toEssImage :=
 instance instEssSurjId : EssSurj (𝟭 C) where
   mem_essImage Y := ⟨Y, ⟨Iso.refl _⟩⟩
 
+theorem Functor.essSurj_of_surj (h : Function.Surjective F.obj) : EssSurj F where
```

Comment:
I am not completely sure we need this lemma.

Reply:
it's already merged either way, see #11239